integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
@duglin 
Thanks for you clarification.
I think I know how to do it.

BTW, I may not be able to update PR before next Thuesday when on the trip.
Also, I would squash the two commits.